cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/AbstractInputsTransformer.java

```diff
@@ -32,7 +32,7 @@ protected Input transformSingleInput(Object rawInput) {
             return new Input(inputName, null);
         } else if (rawInput instanceof Map) {
             Map.Entry<String, ?> entry = ((Map<String, ?>) rawInput).entrySet().iterator().next();
-            Object entryValue = entry.getValue();
+            Serializable entryValue = (Serializable) entry.getValue();
             if(entryValue == null){
                 throw new RuntimeException("Could not transform Input : " + rawInput + " Since it has a null value.\n\nMake sure a value is specified or that indentation is properly done.");
```

Comment:
Do you see a use case for it?
It will anyway be assigned in runtime, if we don't have the right input on the context to bind from...